### ISSUE TITLE
What should be done with the epoch=000XXX.ckpt files?

### ISSUE THREAD

**Issue by aperullo** [rinongal/textual_inversion#27]
Hi, awesome paper. Thanks for making the code available. I wanted to know about the epoch...ckpt that appear in the same folder as the embeddings. They appear a whenever a new minimal val loss is reached, but they can't be used as the ckpt for the stable diffusion model. They're too small to be the weights.

I'd presume I should use the embeddings that correspond to the best rather than the latest. Is there a way to correlate them?

**Comment by rinongal** [comment]
These are actually the checkpoints of the model itself which Pytorch-Lightning helpfully saves during training. However, since we don't want them to actually clog disk space (as most of the model isn't being changed), we're telling it to save them empty. 

In other words, there's really nothing in them and you can disregard them.
